libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ServiceEventConnection.java

```diff
@@ -0,0 +1,36 @@
+package org.wordpress.android.util;
+
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.IBinder;
+
+import org.greenrobot.eventbus.EventBus;
+
+public class ServiceEventConnection {
```

Comment:
What do you think about moving this entire class into `AutoForeground.java`? It's unclear what its purpose is if we leave it alone within a `util` package.

Reply:
Makes sense! Done in 194b576.